src/funding/funding.js

```diff
@@ -62,6 +62,10 @@ export function isFundingEligible(source : $Values<typeof FUNDING>,
         return false;
     }
 
+    if (fundingConfig.requiresPopupSupport === true && supportsPopups() === false) {
```

Comment:
This logic needs to be able to run on the server, so `supportsPopups()` will error. Instead we should pass down `supportsPopups` as a boolean flag into the renderer.